### PR TITLE
Trapi 1.4

### PR DESCRIPTION
**Pull request by YaphetKG**
- Logic for formatting sources attribute for edge in cypher,
- edge bindings returned under analyses key

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="reasoner-transpiler",
-    version="1.11.11",
+    version="2.0.0",
     author="Patrick Wang",
     author_email="patrick@covar.com",
```
Comment (cbizon) [inline setup.py:8]: Let's change the author to  somebody that can still be contacted

**Review by cbizon — COMMENTED** [review]
Would it be possible to add a test that validates conformance to the schema?    Maybe use of pydantic in the transpiler already takes care of that level of validation?

File: setup.py (hunk 1 above)
Comment (YaphetKG) [reply setup.py:8]: Hi Chris, I have added (https://github.com/ranking-agent/reasoner-transpiler/blob/trapi-1.4/setup.py#L9-L10),  based on this (https://docs.python.org/3/distutils/setupscript.html#additional-meta-data)

Comment (YaphetKG) [reply setup.py:8]: i.e maintainer and maintainer_email feilds

**Comment by YaphetKG** [comment]
> 

I was thinking because plater would implement the pydantic models, validation would be assured there after all the extra formatting that needs to happen. But i think we could still do partial validation at this level too.

**Pull request merged**